src/main/java/demo/multiagent/application/AgenticWorkflow.java

```diff
@@ -98,6 +100,7 @@ public AgenticWorkflow(AgentsRegistry agentsRegistry, Selector agentSelector, Pl
   public WorkflowDef<State> definition() {
     return workflow()
       .defaultStepRecoverStrategy(maxRetries(1).failoverTo(INTERRUPT))
+      .defaultStepTimeout(Duration.of(30, SECONDS))
```

Comment:
otherwise the 5 secs default timeout is not enough... and the workflow gets into interrupt state. Maybe Renate had this change locally?